api/service.go

```diff
@@ -354,12 +380,12 @@ func (s *service) Listen(req *pb.ListenRequest, server pb.API_ListenServer) erro
 		case pb.ListenRequest_Filter_SAVE:
 			listenActionType = db.ListenSave
 		default:
-			return status.Errorf(codes.InvalidArgument, "invalid filter action %v", filter.GetAction())
+			return status.Errorf(codes.InvalidArgument, "invalid filter action %v", filter.Action)
 		}
 		options[i] = db.ListenOption{
 			Type:       listenActionType,
-			Collection: filter.GetCollectionName(),
-			ID:         core.InstanceID(filter.InstanceID),
+			Collection: filter.CollectionName,
```

Comment:
What's the difference between accessing properties in the generated code using the "get" method vs the property directly? Never been sure about that.

Reply:
I think they try to return the zero-value of the property _other than nil_. So, in the case of strings, the getter returns `""` instead of nil. But, in the case of types who's zero value _is nil_, they just return `nil`, e.g.,

```
func (m *NewDBFromAddrRequest) GetReadKey() []byte {
	if m != nil {
		return m.ReadKey
	}
	return nil
}
```

I'm not sure this is actually useful though, since we often have cases where nil is a valid value. So, we end up checking anyway. Honestly though, I was just going for consistency.

Reply:
Cool, yea I'd like to be consistent about using one way or another. Using the property sounds good to me.